### PR TITLE
Routing on the client with Redux

### PR DESCRIPTION
**Pull request by borellvi**
#686 
Adding a some basic routing to the Redux example

As @arunoda suggested I've tried to keep it similar to the mobx example.

### CODE REVIEW

File: examples/with-redux/components/Page.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```
Comment (arunoda) [inline examples/with-redux/components/Page.js]: We don't need this line.

**Review by arunoda — CHANGES_REQUESTED** [review]
This is great.
Could you just do the above change ?
Then we are good to go.

**Comment by borellvi** [comment]
@arunoda well spotted, done!

**Review by arunoda — APPROVED** [review]

**Comment by arunoda** [comment]
Awesome.

**Pull request merged**